modules/c++/sys/source/OSWin32.cpp

```diff
@@ -229,6 +229,29 @@ std::string sys::OSWin32::getEnv(const std::string& s) const
     return result;
 }
 
+bool sys::OSWin32::isEnvSet(const std::string& s) const
+{
+    try
```

Comment:
And as long as you're in the code... In the `getEnv()` implementation, can you switch this to a vector rather than a raw pointer?